src/components/SubscriptionList.vue

```diff
@@ -48,7 +54,7 @@
         </td>
         <td>
           <p v-for="entity in subscription.subject.entities">
-            {{entity.id}}, {{entity.type}}
+            {{entity.idPattern}}, {{entity.typePattern}}
```

Comment:
I thought you was going to keep both id and idPattern, considering the subtleties we skyped yesterday. Weren't you?

Reply:
@taliaga I thought at first that I could get away with using just patterns, but in retrospect: if there happen to be IDs that can be substrings of other IDs by accident then it would cause issues. I'll need to fix this.

Reply:
@taliaga Now fixed